Jenkinsfile

```diff
@@ -54,5 +55,24 @@ pipeline {
                 archiveArtifacts artifacts: "$APK_PATH/*.apk", onlyIfSuccessful: true
             }
         }
+        stage('checkout test project repo') {
+            steps {
+                sh "mkdir -p $TEST_PROJECT_REPO_NAME"
```

Comment:
you shouldn't need to create the directory, by using the `dir()` command and a git checkout, it should take care of it.

Reply:
Done